vector/src/main/java/im/vector/app/features/settings/VectorDataStore.kt

```diff
@@ -59,4 +59,16 @@ class VectorDataStore @Inject constructor(
             settings[forceDialPadDisplay] = force
         }
     }
+
+    private val forceLoginFallback = booleanPreferencesKey("force_login_fallback")
```

Comment:
I would have expected this to be beside the dial pad option within the `VectorDataStore` :thinking: 

I have an upcoming PR to move the override options out of the `VectorDataStore` and into a `VectorOverrides` abstraction

Reply:
Soz for the conflict 😅 For visibility sake, because they're pretty minor, we decided that you'd resolve them on your pr

Reply:
Soz for the conflict 😅 For visibility sake, because they're pretty minor, we decided that you'd resolve them on your pr